src/ring_theory/noetherian.lean

```diff
@@ -41,6 +43,7 @@ is proved in `ring_theory.polynomial`.
 ## References
 
 * [M. F. Atiyah and I. G. Macdonald, *Introduction to commutative algebra*][atiyah-macdonald]
+* [P. Samuel, *Théorie algébrique des nombres*][samuel]
```

Comment:
Shouldn't these be added to the `.bib` file instead?
(I guess the Atiyah-Macdonald reference is from before we started the `.bib` file.)

Reply:
Oh, this I really don't know! I realise I have no clue about bibliographical issues, so I'm keen to learn.

Reply:
Just add entries in https://github.com/leanprover-community/mathlib/blob/master/docs/references.bib
(and then cite them at the top of the file using `[bibkey]`).

Reply:
Done! #5018

Reply:
You could of course also have done it in this PR, but it doesn't matter. I've kicked #5018 on the queue.

Reply:
```suggestion
* [samuel]
```